src/app/core/components/event-card/event-card.component.scss

```diff
@@ -0,0 +1,154 @@
+@import 'src/styles-variables.scss';
+
+.event-card {
+  &.pocket {
+    min-width: 240px;
+    .content {
+      .title {
+        @include text-style-h5;
+      }
+      .datetime {
+        @include text-style-body2;
+        display: flex;
+        align-items: center;
+        .mat-icon {
+          margin-right: 1rem;
+        }
+      }
+    }
+  }
+  &.list-small {
+    min-height: 80px;
+
+    width: 100%;
+
+    display: flex;
+    align-items: center;
+
+    padding: 0;
+    overflow: hidden;
+
+    .image-container {
```

Comment:
Some selectors here have same name but being completely assigned different  properties. ex. .image-container, .title, .datetime. Consider add '-' or '_' to name selectors prefix instead of assigning same name to it. Since our teams haven't committed to any CSS naming system yet, you can use your own preferred one. just make it easy to read and consistent.